howdoi/howdoi.py

```diff
@@ -410,29 +409,41 @@ def _get_answers(args):
     if not question_links:
         return False
 
-    answers = []
-    initial_position = args['pos']
-    multiple_answers = (args['num_answers'] > 1 or args['all'])
+    initial_position = args['pos'] - 1
+    final_position = initial_position + args['num_answers']
+    question_links = question_links[initial_position: final_position]
```

Comment:
I think you can remove the whitespace between `initial_position: final_position`

Reply:
I think I'll also change the variable names, line 414 seems a little too wordy